src/documents/ModelTransformationAPI.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Model Transformation API
+layout: default
+category: Infusion
+---
+
+The *** model transformation *** framework is a core part of Infusion and allows you to transform change an input [model](FrameworkConcepts.md#model-objects) (JSON structure) based on a set of rules. 
```

Comment:
This opening sentence should probably say "transform" or "change", but not both ("allows you to transform change an input...")